rel-eng/push-packages-to-obs.sh

```diff
@@ -274,7 +274,7 @@ while read PKG_NAME; do
           sed "s/^ARG URL=.*$/ARG URL=\"https:\/\/www.suse.com\/products\/suse-manager\/\"/" -i $SRPM_PKG_DIR/Dockerfile
           sed "s/^ARG REFERENCE_PREFIX=.*$/ARG REFERENCE_PREFIX=\"registry.suse.com\/suse\/manager\/${VERSION}\"/" -i $SRPM_PKG_DIR/Dockerfile
       fi
-      sed "s/%PKG_VERSION%/${PRODUCT_VERSION}/g" -i $SRPM_PKG_DIR/Dockerfile
+      sed "/^#\!BuildTag:/ s/$/ ${SEMANTIC_VERSION}/" -i $SRPM_PKG_DIR/Dockerfile
```

Comment:
Outcome of this is:
```
#!BuildTag: uyuni/proxy-tftpd:latest uyuni/proxy-tftpd:4.4.2 uyuni/proxy-tftpd:4.4.2.%RELEASE% 2023.1.0
```
`rhn_web.conf` value in this PR is '2023.01`. I guess we don't have a way of having `2023.01` or `2023.01.0` as this needs to be semantic?

Reply:
For SUSE Manager we get `4.4.0-alpha1`, which is what `rhn_web.conf` has for SUSE Manager. Will this work?

Reply:
Finally, I am wondering if it's OK having a difference in the namespaces:
```
#!BuildTag: uyuni/proxy-helm:latest uyuni/proxy-helm:4.4.3 uyuni/proxy-helm:4.4.3-build%RELEASE% 2023.1.0
```
vs
```
#!BuildTag: suse/manager/4.4/proxy-helm:latest suse/manager/4.4/proxy-helm:4.4.3 suse/manager/4.4/proxy-helm:4.4.3-build%RELEASE% 4.4.0-alpha1
```

Major version (`4.4`) is included for SUSE Manager. But we don't have it at all for Uyuni.

Could be normal, as Uyuni doesn't really have a "major version" as changing the Year doesn't mean breaking changes.